lockbox-ios/Common/Extensions/UIViewController+.swift

```diff
@@ -4,6 +4,7 @@
 
 import UIKit
 import RxSwift
+import RxCocoa
 
 protocol StatusAlertView {
```

Comment:
Nit: naming. This shouldn't be called `View` if it isn't a `UIView`.

Suggest: `StatusAlertDisplay`? `InfiniteProgressDisplay`?